cmake/DyninstOptions.cmake

```diff
@@ -13,27 +13,19 @@ option(ENABLE_LTO "Enable Link-Time Optimization" OFF)
 
 option(ENABLE_DEBUGINFOD "Enable debuginfod support" OFF)
 
-set(STERILE_BUILD
-    ON
-    CACHE BOOL "Do not download/build any third-party dependencies from source")
+option(STERILE_BUILD "DEPRECATED -- Do not use" OFF)
 
-set(ADD_VALGRIND_ANNOTATIONS
-    OFF
-    CACHE BOOL "Enable annotations for Valgrind analysis")
+option(ADD_VALGRIND_ANNOTATIONS "Enable annotations for Valgrind analysis" OFF)
 
-set(ENABLE_STATIC_LIBS
-    NO
```

Comment:
OFF instead of NO for consistency?

Reply:
Oops, sorry. I didn't mean to request a review on this. I agree we should use OFF instead of NO. I just copy/pasted.